backstop/tests/menu.html

```diff
@@ -442,7 +442,11 @@ <h2>Skeleton</h2>
           style="--iui-menu-item-content-skeleton-max-width: 30%;"
         >
           <span class="iui-content">
-            <div class="iui-menu-label iui-skeleton"></div>
+            <div
+              class="iui-menu-label iui-skeleton"
+              aria-hidden="true"
+            ></div>
+            <div class="iui-visually-hidden">Loading...</div>
```

Comment:
```suggestion
            <div class="iui-visually-hidden">Loading&mldr;</div>
```
@VirginieGaudreault can you check how `Loading...` and `Loading…` are read via screen readers, is there any difference / benefit?

Reply:
I used NVDA and it read to me as `Loading` no dots or ellipses mentioned.